### ISSUE TITLE
Scenes are not auto appearing on top bar of Foundry game after importing 

### ISSUE THREAD

**Issue by illianne** [ForgeVTT/forge-compendium-browser#31]
- open compendium browser
- choose any scene & import
- navigate to scenes & confirm scene has been imported
➡️ Result: scene is not displayed across top of game
Workaround: refresh game and scene will appear

https://github.com/ForgeVTT/forge-compendium-browser/assets/5169836/e87ec5cd-ac20-4ec9-a441-2c28ef958936

**Comment by kakaroto** [comment]
Good catch, though that sounds like a Foundry bug.  I just tested it on Foundry 11.308 and I'm not seeing the issue

**Comment by illianne** [comment]
✅ Validated fixed scenes are being activated and added to game immediately
✅ Validated imported scenes Set scene as navigatable setting turned OFF
